src/value.rs

```diff
@@ -180,6 +181,7 @@ pub(crate) enum TypeTag {
     I64 = 4,
     U64 = 5,
     InlineInt = 6,
+    InlineString = 7,
```

Comment:
the comment in `ValueType` should be changed to reflect the new TypeTag. (GitHub won't let me comment on non-diff lines)